lrlex/src/lib/ctbuilder.rs

```diff
@@ -547,24 +547,31 @@ where
-                            let path_joined = grm_path.parent().unwrap().join(test_files);
-                            for path in
-                                glob(&path_joined.to_string_lossy()).map_err(|e| e.to_string())?
-                            {
-                                let path = path?;
-                                if let Some(ext) = path.extension() {
-                                    if let Some(ext) = ext.to_str() {
-                                        if ext.starts_with("grm") {
-                                            Err(ErrorString("test_files extensions beginning with `grm` are reserved.".into()))?
+                        Some(HeaderValue(_, Value::Setting(Setting::Array(test_globs, _, _)))) => {
+                            for setting in test_globs {
+                                match setting {
+                                    Setting::String(test_files, _) => {
+                                        let path_joined = grm_path.parent().unwrap().join(test_files);
+                                        for path in
+                                            glob(&path_joined.to_string_lossy()).map_err(|e| e.to_string())?
+                                        {
+                                            let path = path?;
+                                            if let Some(ext) = path.extension() {
+                                                if let Some(ext) = ext.to_str() {
+                                                    if ext.starts_with("grm") {
+                                                        Err(ErrorString("test_files extensions beginning with `grm` are reserved.".into()))?
+                                                    }
+                                                }
+                                            }
+                                            let input = fs::read_to_string(&path)?;
+                                            let l: LRNonStreamingLexer<LexerTypesT> =
+                                                closure_lexerdef.lexer(&input);
+                                            for e in rtpb.parse_map(&l, &|_| (), &|_, _| ()).1 {
+                                                Err(format!("parsing {}: {}", path.display(), e))?
+                                            }
                                         }
                                     }
-                                }
-                                let input = fs::read_to_string(&path)?;
-                                let l: LRNonStreamingLexer<LexerTypesT> =
-                                    closure_lexerdef.lexer(&input);
-                                for e in rtpb.parse_map(&l, &|_| (), &|_, _| ()).1 {
-                                    Err(format!("parsing {}: {}", path.display(), e))?
+                                _ => return Err("Invalid value for setting 'test_files'".into()),
```

Comment:
This `_ =>` branch looks like it is at the wrong indentation level, but for whatever reason rustfmt gives up on it here.